src/controllers/queries/index.ts

```diff
@@ -224,13 +224,79 @@ export class QueriesController extends HTMLController {
     }
 
     return this.html(
-      this.scope3CarbonConsumptionResponseTemplates.newScope3CarbonConsumptionResponseFormPage(
-        'form',
-        connection,
+      this.scope3CarbonConsumptionResponseTemplates.newScope3CarbonConsumptionResponseFormPage({
+        formStage: 'form',
+        company: connection,
         queryId,
-        query.details['quantity'],
-        query.details['productId']
-      )
+        quantity: query.details['quantity'],
+        productId: query.details['productId'],
+      })
+    )
+  }
+
+  /**
+   * @param param.queryId:UUID - query uuid identifier
+   * @param param.companyId:UUID - connection uuid identifier
+   * @param query.partialQuery:'on' - either render partial or not (checkbox state)
+   *
+   * @returns a table of connections for partial query
+   */
+  @SuccessResponse(200)
+  @Get('/{queryId}/partial/{companyId}')
+  public async scope3CO2Partial(
+    @Path() queryId: UUID,
+    @Path() companyId: UUID,
+    @Query() partialQuery?: 'on'
```

Comment:
alright what's the deal with `partialQuery` only having one valid value of `on`. Surely this is unnecessary?

Reply:
Basically, we are using checkbox input and by the default when it's checked it sends a query string `partialQuery = 'on'` following the `http://localhost:3000/form?<checkbox_name>=on`, happy to rename or... not sure, but this way I can do dynamic tables and re-use same page form

Reply:
no this is reasonable logic. Maybe add a comment just to explain this

Reply:
There was one already at the top of the method. Updated it to be a bit more insightful
line 229 - `* @param query.partialQuery:'on' - either render partial or not (checkbox state)` **changed to ->**
```
   * @param query.partialQuery:'on' - either render partial. Checkbox state for checked it will
   * add query string to the URL using the name of the input (checkbox)
```

Reply:
@mattdean-digicatapult There was one already at the top of the method. Updated it to be a bit more insightful
line 229 - `* @param query.partialQuery:'on' - either render partial or not (checkbox state)` **changed to ->**
```
   * @param query.partialQuery:'on' - either render partial. Checkbox state for checked it will
   * add query string to the URL using the name of the input (checkbox)
```